core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4845,6 +4846,23 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_FILE_METADATA_SYNC_TRAVERSAL_ORDER =
+       enumBuilder(Name.USER_FILE_METADATA_SYNC_TRAVERSAL_ORDER,
+           MetadataSyncTraversalOrder.class)
+          .setDefaultValue(MetadataSyncTraversalOrder.BFS)
+          .setDescription("The pending Path in the Inode SyncStream traversal order, DFS consumes"
+              + " less memory while BFS is more fair for all concurrent sync tasks. For more"
+              + " description see the comments of MasterMetadataSyncTraverseType")
+          .setScope(Scope.CLIENT)
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .build();
```

Comment:
```suggestion
  public static final PropertyKey MASTER_METADATA_SYNC_TRAVERSAL_ORDER =
       enumBuilder(Name.USER_FILE_METADATA_SYNC_TRAVERSAL_ORDER,
           MetadataSyncTraversalOrder.class)
          .setDefaultValue(MetadataSyncTraversalOrder.BFS)
          .setDescription("The pending Path in the Inode SyncStream traversal order, DFS consumes"
              + " less memory while BFS is more fair for all concurrent sync tasks. For more"
              + " description see the comments of MasterMetadataSyncTraverseType")
          .setScope(Scope.MASTER)
          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
          .build();
```

Reply:
Per discussion, changing this to a Scope.USER property involves passing this value from the user side, which involves updating existing RPC. Configuring sync order per path is more like micro management and should not be controlled by the user. 

So let's change it back to master side property. Thanks!